website/versioned_docs/version-0.62/rnw-dependencies.md

```diff
@@ -0,0 +1,51 @@
+---
+id: version-0.62-rnw-dependencies
+title: System Requirements
+original_id: rnw-dependencies
+---
+
+To check or install dependencies you'll need to build RNW and RNW apps, run the script [rnw-dependencies.ps1](https://github.com/microsoft/react-native-windows/blob/master/vnext/Scripts/rnw-dependencies.ps1) in an elevated PowerShell window.
```

Comment:
The top section here is not needed - it is now part of the first subsection below. This page should start with the title "## Windows Development Dependencies"

Reply:
Updated!